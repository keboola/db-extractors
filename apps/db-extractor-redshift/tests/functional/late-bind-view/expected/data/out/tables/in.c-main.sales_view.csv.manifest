{
    "destination": "in.c-main.sales_view",
    "incremental": false
}