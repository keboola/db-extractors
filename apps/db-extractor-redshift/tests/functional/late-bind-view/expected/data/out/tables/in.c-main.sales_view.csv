"usercity","zipcode"
"Mize","39153"
"The Lakes","89124"
"Baldwin","21020"
"Archbald","18501"
"Berea","44127"
"Baldwin","21219"
"Phoenix","85083"
"Martinsburg","25428"
"Los Angeles","91328"
"Bolton","28455"